doc/containers.md

```diff
@@ -77,6 +77,9 @@ security.syscalls.blacklist             | string    | -                 | no
 security.syscalls.blacklist\_compat     | boolean   | false             | no            | container\_syscall\_filtering        | On x86\_64 this enables blocking of compat\_\* syscalls, it is a no-op on other arches
 security.syscalls.blacklist\_default    | boolean   | true              | no            | container\_syscall\_filtering        | Enables the default syscall blacklist
 security.syscalls.whitelist             | string    | -                 | no            | container\_syscall\_filtering        | A '\n' separated list of syscalls to whitelist (mutually exclusive with security.syscalls.blacklist\*)
+snapshots.schedule                      | string    | -                 | no            |                                      | Cron expression (<hour> <dom> <month> <dow>)
```

Comment:
The bits inside the parenthesis will need some escaping so that they show up in rendered markdown